src/mam4xx/ndrop.hpp

```diff
@@ -15,6 +15,21 @@ using Real = haero::Real;
 
 namespace mam4 {
 
+namespace ndrop {
+//TODO: put this inside ndrop_init
+// abdul-razzak functions of width
+Real bizarro1[AeroConfig::num_modes()];
+// abdul-razzak functions of width
+Real bizarro2[AeroConfig::num_modes()];
```

Comment:
these are f1 and f2, but getting a weird error that mam4::f1 already exists? 
```
libmam4xx_tests.a(kohler_verification.cpp.o):(.bss+0x0): multiple definition of `mam4::ndrop::bizarro1'
CMakeFiles/kohler_unit_tests.dir/kohler_unit_tests.cpp.o:(.bss+0x0): first defined here
libmam4xx_tests.a(kohler_verification.cpp.o): In function `__gnu_cxx::char_traits<char>::eq(char const&, char const&)':
/people/litz372/mam4/mam4xx/src/tests/kohler_verification.cpp:17: multiple definition of `mam4::ndrop::bizarro2'
CMakeFiles/kohler_unit_tests.dir/kohler_unit_tests.cpp.o:/people/litz372/mam4/mam4xx/haero_cpu_double_debug/include/spdlog/pattern_formatter-inl.h:160: first defined here
collect2: error: ld returned 1 exit status
```

Reply:
If you want to have these be global variables, you should declare them as `static` so that different `.cpp` files don't all define them separately. That's what's happening to produce this error.

It's probably better not to have these live in the header file, or to attach them to an instance like your comment suggests.

Reply:
Ah, perhaps something to live in the NDrop class?